sos/report/plugins/hbn.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+
+
+class HBN(Plugin, IndependentPlugin):
+
+    short_desc = 'HBN (Host Based Network)'
+    plugin_name = "hbn"
```

Comment:
this intended to be ran inside the container image and not on the DPU?
if that is the case I think it should depend on some package

Reply:
In general i agree but inspecting the pod i saw that all the paths are mounted 
```
      {
        "containerPath": "/etc/cumulus",
        "gidMappings": [],
        "hostPath": "/var/lib/hbn/etc/cumulus",
...
      {
        "containerPath": "/var/log/hbn",
        "gidMappings": [],
        "hostPath": "/var/log/doca/hbn",
```